src/app/components/list-data-by-cagories.tsx

```
@@ -143,48 +143,50 @@ export default function ListDataByCategories(props: {
 				{generalCategory.map((categoryName, categoryIndex) => {
-					return (
-						<div key={`category-${categoryIndex}`}>
-							<div
-								className={`text-sm font-bold ${focused !== "" && focused.split("-")[0] !== `${categoryIndex}` && "opacity-20"} `}
-							>
-								{categoryName}
-							</div>
-							<table className="w-full table-auto">
-								<thead
-									className={`${focused !== "" && focused.split("-")[0] !== `${categoryIndex}` && "opacity-20"}`}
+					if (categoryGorupMap.get(categoryName)?.length || 0 > 0) {
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)